src/QMCWaveFunctions/EinsplineSetBuilderOld.cpp

```diff
@@ -37,20 +26,26 @@ bool EinsplineSetBuilder::ReadOrbitalInfo(bool skipChecks)
     return false;
   }
 
-  // Read format
-  std::string format;
-  H5File.read(format, "/format");
-  H5File.read(Version, "/version");
-  app_log() << "  HDF5 orbital file version " << Version[0] << "." << Version[1] << "." << Version[2] << "\n";
-  if (format.find("ES") < format.size())
+  try
   {
+    // Read format
+    std::string format;
+    H5File.read(format, "/format");
+    if (format.find("ES") == std::string::npos)
+      throw std::runtime_error("Format string input \"" + format + "\" doesn't contain \"ES\" keyword.");
     Format = ESHDF;
-    return ReadOrbitalInfo_ESHDF(skipChecks);
+    H5File.read(Version, "/version");
+    app_log() << "  HDF5 orbital file version " << Version[0] << "." << Version[1] << "." << Version[2] << std::endl;
+  }
+  catch (const std::exception& e)
+  {
+    app_error() << e.what() << std::endl
+                << "EinsplineSetBuilder::ReadOrbitalInfo too old h5 file or it is not a bspline orbital file!"
```

Comment:
Maybe change "too old h5 file" -> "h5 file format is too old"

Reply:
Updated.